src/modules/admin/admin.route.tsx

```
@@ -0,0 +1,34 @@
+  dashboard: `${PATHS.admin}/overview`,
+  product: `${PATHS.admin}/products`,
+  manageUser: `${PATHS.admin}/users`,
+  manageOrder: `${PATHS.admin}/orders`,
+}
+
+export const adminRoutes = [
+  <Route
+    id={adminPaths.dashboard}
+    path={adminPaths.dashboard}
+    element={<Admin />}
+  />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)